crates/hyperswitch_connectors/src/connectors/fiserv.rs

```diff
@@ -44,12 +43,23 @@ use time::OffsetDateTime;
 use transformers as fiserv;
 use uuid::Uuid;
 
-use crate::{constants::headers, types::ResponseRouterData, utils};
+use crate::{
+    constants::headers,
+    types::ResponseRouterData,
+    utils::{construct_not_implemented_error_report, convert_amount},
+};
 
-#[derive(Debug, Clone)]
-pub struct Fiserv;
+#[derive(Clone)]
+pub struct Fiserv {
+    amount_converter: &'static (dyn AmountConvertor<Output = MinorUnit> + Sync),
+}
 
 impl Fiserv {
+    pub fn new() -> &'static Self {
+        &Self {
+            amount_converter: &MinorUnitForConnector,
```

Comment:
```suggestion
            amount_converter: &FloatMajorUnitForConnector,
```
According to the [doc](https://docs.fiserv.dev/public/reference/createpaymenturl), the amount accepted by the connector is in `FloatMajorUnit`. Please make the necessary changes accordingly at all relevant places for the Fiserv connector.